app/controllers/hosts_controller.rb

```diff
@@ -907,4 +907,11 @@ def host_power_ping(result)
   def host_power_state(key)
     HOST_POWER[key].merge(:title => _(HOST_POWER[key][:title]))
   end
+
+  def host_attributes_for_templates(host)
+    host_params.select do |k,v|
+      host.class.new.attributes.include?(k) &&
```

Comment:
No need to do it on every param, that's for sure, thanks for the catch 😄 I can get the list of attributes beforehand.

The reason is that when I have a `Host::Discovered`, there may be parameters on `host_params` that I don't want to use. E.g: `is_owned_by`: is not defined by Host::Discovered, so those parameters I want to skip.